### PR TITLE
Minor ortographic corrections of ruletypes.h

### PR DESCRIPTION
**Pull request by thalic**
Minor ortographic corrections of ruletypes.h

### CODE REVIEW

File: common/ruletypes.h [hunk 1]
```diff
@@ -147,12 +147,12 @@ RULE_BOOL(Character, ActiveInvSnapshots, false, "Takes a periodic snapshot of in
...
 RULE_INT(Character, InvSnapshotHistoryD, 30, "Time to keep snapshot entries (days)")
-RULE_BOOL(Character, RestrictSpellScribing, false, "Setting whether to testricts spell scribing to allowable races/classes of spell scroll")
+RULE_BOOL(Character, RestrictSpellScribing, false, "Setting whether to restrict spell scribing to allowable races/classes of spell scroll")
 RULE_BOOL(Character, UseStackablePickPocketing, true, "Allows stackable pickpocketed items to stack instead of only being allowed in empty inventory slots")
 RULE_BOOL(Character, EnableAvoidanceCap, false, "Setting whether the avoidance cap should be activated")
 RULE_INT(Character, AvoidanceCap, 750, "750 Is a pretty good value, seen people dodge all attacks beyond 1,000 Avoidance")
 RULE_BOOL(Character, AllowMQTarget, false, "Disables putting players in the 'hackers' list for targeting beyond the clip plane or attempting to target something untargetable")
-RULE_BOOL(Character, UseOldBindWound, false, "Uses the original bind wound behavior")
+RULE_BOOL(Character, UseOldBindWound, false, "Uses the original bind wound behaviour")
```
Comment (joligario) [inline common/ruletypes.h:155]: EN-US is fine here.

File: common/ruletypes.h [hunk 2]
```diff
@@ -191,13 +191,13 @@ RULE_CATEGORY_END()
 
 RULE_CATEGORY(Skills)
 RULE_INT(Skills, MaxTrainTradeskills, 21, "Highest level for trading skills that can be learnt by the trainer")
-RULE_BOOL(Skills, UseLimitTradeskillSearchSkillDiff, true, "Enables the limit for the maximum difference between trivial and skill for recipe searches and favorites")
+RULE_BOOL(Skills, UseLimitTradeskillSearchSkillDiff, true, "Enables the limit for the maximum difference between trivial and skill for recipe searches and favourites")
```
Comment (joligario) [inline common/ruletypes.h:194]: EN-US is fine here

File: common/ruletypes.h [hunk 3]
```diff
@@ -508,35 +508,35 @@ RULE_INT(NPC, MajorNPCCorpseDecayTimeMS, 1500000, "NPC corpse decay time, if NPC
...
-RULE_BOOL(NPC, UseClassAsLastName, true, "Uses class archetype as LastName for NPCwith none")
+RULE_BOOL(NPC, UseClassAsLastName, true, "Uses class archetype as LastName for NPC with none")
 RULE_BOOL(NPC, NewLevelScaling, true, "Better level scaling, use old if new formulas would break your server")
 RULE_INT(NPC, NPCGatePercent, 20, " Percentage at which the NPC Will attempt to gate at")
 RULE_BOOL(NPC, NPCGateNearBind, false, "Will NPC attempt to gate when near bind location?")
 RULE_INT(NPC, NPCGateDistanceBind, 75, "Distance from bind before NPC will attempt to gate")
 RULE_BOOL(NPC, NPCHealOnGate, true, "Will the NPC Heal on Gate")
 RULE_BOOL(NPC, UseMeditateBasedManaRegen, false, "Based NPC ooc regen on Meditate skill")
-RULE_REAL(NPC, NPCHealOnGateAmount, 25, "How much the npc will heal on gate if enabled")
+RULE_REAL(NPC, NPCHealOnGateAmount, 25, "How much the NCP will heal on gate if enabled")
```
Comment (joligario) [inline common/ruletypes.h:532]: NPC

**Comment by thalic** [comment]
Requested changes implemented.

**Review by joligario — APPROVED** [review]

**Review by Akkadius — APPROVED** [review]

**Pull request merged**